public/app/features/expressions/components/Reduce.tsx

```diff
@@ -66,8 +72,30 @@ export const Reduce = ({ labelWidth = 'auto', onChange, refIds, query }: Props)
     );
   };
 
+  const strictModeNotification = () => {
+    if (mode !== ReducerMode.Strict) {
+      return null;
+    }
+    return (
+      <Alert title={t('reduce.strictMode.title', 'Strict Mode Behaviour')} severity="info">
+        <Trans i18nKey="reduce.strictMode.description">
```

Comment:
Let's split this into two parts. 
First should hold only the text till the url, and you can put this in json file too. I'll put a comment into the json too. 

```
When <code>Reduce Strict mode</code> is used, the <code>fill(null)</code> function (InfluxQL) will result in{' '}<code>NaN</code>.
```

and the second part should only hold the text inside the `a` tag. With this way when it's translated we won't lose the link. 

```
See the documentation for more details.
```